entrypoint.sh

```diff
@@ -113,8 +113,12 @@ create_uesr_data () {
     # ENCODED_USER_DATA=$(echo "$USER_DATA" | base64 | tr -d \\n)
     cat <<EOF > user_data.sh
 #!/bin/bash
-apt-get update
-apt-get install -y curl jq
+if command -v yum &> /dev/null; then
+    yum install -y curl jq libicu
```

Comment:
do we need yum update here?

Reply:
no, this command can install the libraries